src/Open3D/Geometry/HalfEdgeTriangleMesh.cpp

```diff
@@ -282,6 +282,16 @@ Eigen::Vector3d HalfEdgeTriangleMesh::GetMaxBound() const {
     return Eigen::Vector3d((*itr_x)(0), (*itr_y)(1), (*itr_z)(2));
 }
 
+Eigen::Vector3d HalfEdgeTriangleMesh::GetCenter() const {
+    Eigen::Vector3d center(0, 0, 0);
+    if (!HasVertices()) {
+        return center;
+    }
+    center = std::accumulate(vertices_.begin(), vertices_.end(), center);
```

Comment:
Would not be faster to use Eigen vectorization with colwise operations?

Reply:
Could you provide an example on how to map `std::vector<Eigen::Vector3d>` to `Eigen::Map<Eigen::MatrixXd>` ?

Reply:
I tried the code below, but not sure if the mapping is 100% correct.

```
Eigen::Map<Eigen::Matrix<double, Eigen::Dynamic, 3>> points((double*)points_.data(), points_.size(), 3);
center = points.colwise().mean()
```

However, given this code the mean of 1e8 points is computed in `0.1582[s]` (avg of 20 reps). In the current version it took `0.1636[s]`. The difference does not seem to be too big.